### PR TITLE
20200430 1737

### PR DESCRIPTION
**Pull request by nbhasin2**

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -318,7 +318,7 @@
         <source>$(MOZ_PRODUCT_NAME)</source>
       </trans-unit>
       <trans-unit id="CFBundleShortVersionString">
-        <source>23.1</source>
+        <source>0.0.1</source>
```
Comment (flodolo) [inline templates/firefox-ios.xliff:321]: This doesn't look right?

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -2796,6 +2827,14 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
     </header>
     <body>
+      <trans-unit id="Intro.Slides.Automatic.Privacy.Description">
+        <source>Privacy by default blocks malware and stops trackers.</source>
```
Comment (flodolo) [inline templates/firefox-ios.xliff:2831]: I don't understand this sentence. What's the subject here? Is it "Privacy" or "Privacy by default"?

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (nbhasin2) [reply templates/firefox-ios.xliff:321]: As we don't update version number on master branch we decided to make it constant 0.0.1 for master.

Hence why we have this change.

File: templates/firefox-ios.xliff (hunk 2 above)
Comment (nbhasin2) [reply templates/firefox-ios.xliff:2831]: > I don't understand this sentence. What's the subject here? Is it "Privacy" or "Privacy by default"?

This is the description for using a browser that respects privacy of their users at all times as in Privacy by default.

Ref UI - 
https://app.abstract.com/share/fb3066fc-2401-48a1-951d-609eb81c3913?collectionId=912f4b7f-895d-40ec-b7c9-0a2c9cd5af8e&collectionLayerId=e965dff4-deb4-47fd-96dc-b10cc050233c

Comment (flodolo) [reply templates/firefox-ios.xliff:2831]: It's still unclear to me what "Privacy by default" stands for: should it be intended as a sort of feature name, does it refer to the default privacy setting? Right now it's really hard to translate.

@betsymi
Any chance you could shed some light?

Comment (betsymi) [reply templates/firefox-ios.xliff:2831]: Not a feature name, no. It refers to our default privacy settings. 

If this is too confusing to translate, we can adjust to `Enhanced Tracking Protection blocks malware and stops trackers.`

Comment (flodolo) [reply templates/firefox-ios.xliff:2831]: I think that would help translators a lot.

**Comment by nbhasin2** [comment]
@flodolo Updated the changes in the following PR hence closing this one

https://github.com/mozilla-l10n/firefoxios-l10n/pull/93

**Pull request closed without merging**